openedx_events/learning/data.py

```diff
@@ -486,10 +513,87 @@ class ORASubmissionData:
         answer (ORASubmissionAnswer): Answer submitted by the user in the ORA submission.
     """
 
+    id = attr.ib(type=str)
+    file_downloads = attr.ib(type=List[ORAFileDownloadsData], factory=list)
```

Comment:
This change doesn't belong here. Could you rebase with main?

Reply:
@kyrylo-kh ^